sources/FSUtils.cpp

```diff
@@ -77,12 +81,13 @@ status_t CheckCopiable(BEntry* src, BEntry* dest)
 		// We have an existing file, so query the user what to do.
 		status_t returncode;
 
-		newpath = "The file ";
-		newpath += name;
-		newpath += " exists. Do you want to replace it?";
+		newpath = B_TRANSLATE_COMMENT("%name% already exists. Do you want to "
```

Comment:
I removed `"The file "` to make it consistent with `ConflictWindow`. I will add it in both windows and also single quote the file name in case there is a space in the name.